app/views/gardens/_thumbnail.html.haml

```diff
@@ -1,36 +1,21 @@
 .panel.panel-success
   .panel-heading
     %h3.panel-title
-      = link_to "#{garden.owner.login_name}'s garden", garden
+      = link_to garden.name, garden
       - if can? :edit, garden
         %a.pull-right{:href => edit_garden_path(garden), :role => "button", :id => "edit_garden_glyphicon"}
           %span.glyphicon.glyphicon-pencil{:title => "Edit"}
-  .panel-body{:id => "gardens_panel_body"}
+  .panel-body{:class => "gardens_panel_body"}
     .row
       .col-md-4
         = link_to image_tag((garden.default_photo ?  garden.default_photo.thumbnail_url : 'placeholder_150.png'), :alt => garden.name, :class => 'img'), garden
       .col-md-8
-        %dl.dl-horizontal
-          %dt Name :
-          %dd= link_to garden.name, garden
-          %dt Location :
+        %dl
+          - if garden.location.present?
+            %dt Location:
```

Comment:
I do prefer "Foo:" to "Foo :" very much, we should keep this - or even drop the ":" and rely on CSS to convey visual seperation